internal/add/tui.go

```
@@ -0,0 +1,1036 @@
+	manualInput textinput.Model
+	manualErr   string
+
+	// edit (also reused by confirm).
+	editFields  editFields
+	editFocus   int // 0=Name 1=Category 2=Group
+	editErr     string
+
+	// cloning.
+	queue       []editFields // resolved selections waiting to clone
+	currentIdx  int          // index into queue
+	currentName string       // for spinner header
```

golangci-lint (2.11.4)

[error] field currentName is unused

(unused)